data/editor.config

```diff
@@ -363,86 +358,81 @@
         <include field="operator" />
       </type>
       <type id="amenity-car_wash">
-        <include group="poi" />
-        <include field="internet" />
+        <include group="poi_internet" />
       </type>
       <type id="amenity-childcare">
-        <include group="poi" />
-        <include field="internet" />
+        <include group="poi_internet" />
       </type>
       <type id="amenity-veterinary" group="health">
-        <include group="poi" />
-        <include field="internet" />
+        <include group="poi_internet" />
       </type>
       <type id="amenity-charging_station">
         <include group="poi" />
         <include field="operator" />
       </type>
       <type id="craft" can_add="no">
-        <include group="poi" />
+        <include group="poi_internet" />
       </type>
       <type id="craft-beekeeper">
-        <include group="poi" />
+        <include group="poi_internet" />
       </type>
       <type id="craft-blacksmith">
-        <include group="poi" />
+        <include group="poi_internet" />
       </type>
       <type id="craft-brewery" group="shop">
```

Comment:
Why only some craft= are marked with "shop"?

Reply:
🤷‍♂️ I don't know.
Where are used groups in OM ? Is it groups from Wiki OSM ?

Reply:
Groups are collections of some tags, they are defined at the beginning of this file.

Does it make sense to mark remaining shop-like crafts with "shop"?

Reply:
group=shop, accomodation, ... is not defined in the start of the file but we can add this group=shop for all craft POI :)
Normally each crafters have shops or part of her house dedicated for her work